README.md

```diff
@@ -34,6 +34,17 @@ You can also package with pipenv to test the CLI tool.
 Run with example RMRs
 1. `pipenv run rct229 evaluate examples\user_rmr.json examples\baseline_rmr.json examples\proposed_rmr.json`
 
+## Developer Best Practices
+
+#### Unit Testing
+- Pytest is used for unit testing: https://docs.pytest.org/en/stable/contents.html
+- A test file should have the same name as the corresponding code file with "_test" appended before the extension.
+- Keep the test file in the same directory as the code file.
```

Comment:
In my experience, a more common practice is to have a parallel folder called unittest or testing containing the test files. It doesn't matter that much to me but the folder could end up having a lot of files.

Reply:
Our thinking was that having the test and code files next to each other is more convenient during development and that we can easily move them into parallel folders when we're done.

Reply:
That sounds fine.